Code_Exercises/Exercise_02_First_Kernel/solution.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * SYCL Quick Reference
+ * ~~~~~~~~~~~~~~~~~~~~
+ *
+ * // Include SYCL header
+ * #include <CL/sycl.hpp>
+ *
+ * // Default construct a queue
+ * auto q = sycl::queue{};
+ *
+ * // Allocate device memory
+ * auto * devPtr = malloc_device<T>(mycount, q);
+ *
+ * // Memcpy
+ * auto event1 = q.memcpy(dst, src, sizeof(T)*n, {dep_event1, dep_event2});
```

Comment:
I felt like the dependency part made this a bit more confusing, it might be better to suggest using wait calls instead of the depdency events?

Reply:
Sure will do